examples/dashboard.yml

```diff
@@ -0,0 +1,100 @@
+sources:
+  nyc_taxi:
+    description: Large version of nyc taxi dataset
+    driver: parquet
+    args:
+      urlpath: '{{ CATLOG_DIR }}data/nyc_taxi_wide.parq'
+      columns: ['dropoff_x','dropoff_y','pickup_x','pickup_y',
+                'dropoff_hour','pickup_hour','passenger_count']
+    metadata:
+      fields:
+        dropoff_hour:
+          label: Drop-off Hour
+        pickup_hour:
+          label: Pick-up Hour
+        fare_amount:
+          label: Fare
+          unit: $
+        passenger_count:
+          label: Passenger Count
+        counts:
+          label: Ride counts
+      plot:
+        xlim: !!python/tuple [-8240227.037, -8231283.905]
```

Comment:
I think I've arranged it so you can simply provide lists and they are cast appropriately.

Reply:
Actually I'm probably wrong about that, I think that was explicitly for dimension ranges.

Reply:
Would be good to provide support for such casting if you can, to avoid this awkwardness.

Reply:
Oh, it's definitely YAML's fault, or at least JSON's fault (from which YAML inherits its scope).  But the fact remains that it's awkward for Python users to write code like the above, so we should probably accept a list as well as a tuple unless we are already using the list/tuple distinction for some semantic purpose in this case.

Reply:
Fixed in https://github.com/pyviz/hvplot/pull/248.